app/views/games/play.html.slim

```diff
@@ -11,3 +11,10 @@ br
 = link_to 'Undo Round', undo_round_game_path(@game), class: 'btn btn-primary', method: 'delete'
 br
 = link_to 'Finish Game', finish_game_path(@game), class: 'btn btn-primary', method: 'patch'
+
+coffee:
+  ($ window).keypress (e) ->
+    if e.keyCode == 0 or e.keyCode == 32
+      e.preventDefault()
+      ($ "#next-button").click()
```

Comment:
Hmm, I see your point.

Read this: http://engineering.freeagent.com/2012/03/23/coffeescript-with-jquery-sprinkles/ and let me know what you think.

Specifically, this makes me vote for `($ something)`: "This is actually a precedence issue, and the idiomatic CoffeeScript way to resolve this is to lose the brackets around the argument, but add them in around the operation you want to take precendence."